deep-learning/_posts/2017-03-27-layer-normalization.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: review
+title: Layer Normalization
+tags: deep-learning essentials 
+cite:
+    authors: "Ba, J.L., Kiros, J.R. and Hinton, G.E"
+    title:   "Layer Normalization"
+    venue:   "NIPS 2016 Deep Learning Symposium"
+pdf: "https://arxiv.org/pdf/1607.06450.pdf"
+---
+
+Layer Normalization is a simple method used to reduce training time in deep neural networks. It can also stabilize the hidden state dynamics in recurrent networks. It is similar to Batch Normalization, but is not dependent on mini-batch size, and performs the same computation at training and testing time.
+
+
+# How it works
+Layer normalization works by computing the normalization statistics (mean and variance) from all of the summed inputs to the neurons within a hidden layer (on a single training case).
+
+# Background
+
+A feed-forward neural network is a non-linear mapping from a input pattern $$\mathbf x$$ to an output vector $$y$$.
```

Comment:
Exactly, the notation is necessary for the rest to make sense, but I'll drop the text and summarize it in a better way.